configure.ac

```diff
@@ -2519,3 +2519,118 @@ https://trac.torproject.org/projects/tor/wiki/doc/TorFragileHardening
 fi
 
 AC_OUTPUT
+
+#
+# Mini-report on what will be built.
+#
+
+PPRINT_INIT
+PPRINT_SET_INDENT(1)
+PPRINT_SET_TS(38)
+
+AS_ECHO
+AS_ECHO("${PPRINT_COLOR_BLDBLU}Tor $PACKAGE_VERSION $PPRINT_COLOR_RST")
+AS_ECHO
+
+PPRINT_SUBTITLE([Build Features])
+
+test "x$enable_gcc_warnings" != "xno" && value=1 || value=0
+PPRINT_PROP_BOOL([GCC Warnings], $value)
```

Comment:
We should call this "verbose warnings"; it applies to clang too.

Reply:
Maybe it would be a good idea to also print, for this setting and all the others, how to override it?

Reply:
Yup. There is one fixup commit that fixes it all. I recommend you rerun the configure script to see the end result. I changed many things from your review :).